### ISSUE TITLE
TensorFlow2.3.0 corresponds to cuda version 10.1

### ISSUE THREAD

**Issue by TriBall3** [HaojHuang/Equivariant-Transporter-Net#3]
TensorFlow2.3.0 corresponds to cuda 10.1, but pytorch1.8.1 corresponds to cuda 10.2. Does this result in testing using CPU instead of gpu?

![image](https://user-images.githubusercontent.com/75004709/161902813-3cf72348-e6f4-4cfb-ba49-a1658ad1a538.png)

`conda install pytorch==1.8.1 torchvision==0.9.1 torchaudio==0.8.1 cudatoolkit=10.2 -c pytorch`

> pybullet build time: Mar 26 2022 03:00:15
2022-04-06 12:59:08.046104: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2022-04-06 12:59:08.053577: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-04-06 12:59:08.054386: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: Quadro RTX 8000 computeCapability: 7.5
coreClock: 1.77GHz coreCount: 72 deviceMemorySize: 47.46GiB deviceMemoryBandwidth: 625.94GiB/s
2022-04-06 12:59:08.054535: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-04-06 12:59:08.054721: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 1 with properties: 
pciBusID: 0000:02:00.0 name: Quadro RTX 8000 computeCapability: 7.5
coreClock: 1.77GHz coreCount: 72 deviceMemorySize: 47.46GiB deviceMemoryBandwidth: 625.94GiB/s
2022-04-06 12:59:08.054852: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /home/fty98/.conda/envs/ravens/lib/python3.7/site-packages/cv2/../../lib64:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin::/usr/local/cuda/lib64
2022-04-06 12:59:08.054927: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2022-04-06 12:59:08.054970: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2022-04-06 12:59:08.054999: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2022-04-06 12:59:08.055033: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2022-04-06 12:59:08.055058: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2022-04-06 12:59:08.055176: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudnn.so.7'; dlerror: libcudnn.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /home/fty98/.conda/envs/ravens/lib/python3.7/site-packages/cv2/../../lib64:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin:/home/fty98/.mujoco/mujoco200/bin::/usr/local/cuda/lib64
2022-04-06 12:59:08.055197: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
**No GPUs detected. Running with CPU.**

**Comment by HaojHuang** [comment]
It just needs PyTorch with Cuda to do the training and testing. TensorFlow is almost no use here. (I just used its tensorboard to record the loss). Simply installing TensorFlow with CPU is fine.

**Comment by HaojHuang** [comment]
I will remove TensorFlow from the environment in a later update.
